PyRDF/__init__.py

```diff
@@ -155,12 +146,11 @@ def include_headers(headers_paths):
         for path_string in headers_paths:
             headers_to_include.update(_get_paths_set_from_string(path_string))
 
-    # If not on the local backend, distribute files to executors
-    if not isinstance(current_backend, Local):
-        current_backend.distribute_files(headers_to_include)
+    # Distribute files to executors
+    current_backend.distribute_files(headers_to_include)
 
     # Declare the headers in ROOT
-    Utils.declare_headers(headers_to_include)
+    Utils.Utils.declare_headers(headers_to_include)
```

Comment:
`Utils.Utils`?

Reply:
Part of the weird naming scheme in PyRDF:
```
PyRDF/
    backend/
        Utils.py
```
In Utils.py
```
...
class Utils
...
```

Reply:
Similarly for `Dist` class in Dist.py, `Backend` class in `Backend.py` . This needs to be addressed imho

Reply:
In general we'd need to adhere to some naming scheme, in this case and many others in the PR I've used the google styleguide, the same used for the docs. It says that only modules should be imported, not class names directly https://google.github.io/styleguide/pyguide.html#Imports

Reply:
Yes, in general we should try to prevent those repetitions, but we can do that in another PR.